src/components/DailyReportFormUpdate/index.js

```diff
@@ -1,212 +1,262 @@
-                      onChange={handleChange}
-                    />
-
-                    <TextArea
-                      type="textarea"
-                      label="Plan for next day"
-                      name="plan"
-                      defaultValue={values.plan}
-                      error={touched.plan && errors.plan}
-                      onChange={handleChange}
-                    />
-
-                    <TextArea
-                      type="textarea"
-                      label="Comment"
-                      name="comment"
-                      defaultValue={values.comment}
-                      error={touched.comment && errors.comment}
-                      onChange={handleChange}
-                    />
-
-                    <Message
-                      success
-                      header="Update Successfully!"
-                      content="Your report has been updated."
-                    />
-
-                    <AsyncButton buttonName="Update" type="submit" loading={loading} />
-                  </Form>
-                )}
+          if (error) return <ErrorMessage error={error} />;
+
+          return (
+            <Form
+              onSubmit={async e => {
+                if (!this.canBeSubmitted()) {
+                  e.preventDefault();
+                  return;
+                }
+                try {
+                  await updateDailyReport();
+                  this.setStatus(true);
+                } catch (e) {
+                  this.setStatus(false);
+                }
+              }}
+              loading={loading}
+              success={success}
+            >
+              <TextInput
+                type="text"
+                label="Title"
+                name="title"
+                value={title}
+                onChange={this.handleTextChange}
+                onBlur={this.handleBlur('title')}
+                error={shouldMarkError('title') ? errors.title : false}
+              />
+
+              <RadioInput
+                onRadioChange={this.handleRadioChange}
+                emotion={emotion}
+              />
+
+              <CreateTaskForm
+                tasks={tasks}
+                handleTaskChange={this.handleTaskChange}
+                handleAddNewTask={this.handleAddNewTask}
+                handleRemoveTask={this.handleRemoveTask}
+                errors={errors}
+              />
+
+              <TextArea
+                type="textarea"
+                label="Plan for next day"
+                name="plan"
+                value={plan}
+                onChange={this.handleTextChange}
+                onBlur={this.handleBlur('plan')}
+                error={shouldMarkError('plan') ? errors.plan : false}
               />
-            </CenterWrapper>
-          )}
-        </Mutation>
-      );
-    }}
-  </Query>
-);
+
+              {comment !== '' && comment !== null ? (
```

Comment:
Any reason to avoid using `lodash/isEmpty`? ( Maybe my question is silly )
{ !isEmpty(comment) ? ( ... ) : ( ... ) }

Reply:
Yea, we can. Maybe i forgot at that moment :)